west.yml

```diff
@@ -14,4 +14,41 @@ manifest:
       remote: ncs
       repo-path: sdk-nrf
       revision: main
-      import: true
+      import:
+        name-blocklist:
+         - zephyr
+    - name: zephyr
+      repo-path: sdk-zephyr
+      revision: pull/1778/head
+      remote: ncs
+      import:
+        name-allowlist:
```

Comment:
not sure I understand.

Why do you want top point specifically to https://github.com/nrfconnect/sdk-zephyr/pull/1778

That PR has already been merged into sdk-nrf/main, and thus the existing `import: true` will ensure the `sdk-zephyr` pointed to by `sdk-nrf/main:west.yml` will also be used in the `ncs-example-application`.

Reply:
this was to test it before it was merged, will update and remove this part

Reply:
> the commit clearly stated this

if you refer to `REVERT` in the title: `[nrf noup] REVERT: Switch to zephyr PR`  
then I must admit I thought this commit was intended to revert something else in this repo, and not that the commit itself was to be removed before merging.

Using a word like `REVERT` in this context may not be obvious to everyone.
Better to write a full sentence that explains that the commit is to be ignored and why.